packages/prisma/README.md

```diff
@@ -20,10 +20,11 @@ You can find the Prisma schema in the docs at [next-auth.js.org/adapters/prisma]
 
 ## Getting Started
 
-1. Install `next-auth` and `@next-auth/prisma-adapter`
+1. Install `next-auth` and `@next-auth/prisma-adapter` as well as `prisma` and `@prisma/client`.
```

Comment:
I think `prisma` and `@prisma/client` are the same. 🤔

Reply:
I don't think so, [`prisma`](https://www.npmjs.com/package/prisma) is the whole package (incl. introspection, studio, etc. code) and [`@prisma/client`](https://www.npmjs.com/package/@prisma/client) is the auto-generated client part based upon your schema, as far as I understand..